src/harness/BenchmarkDotNet.Extensions/ExclusionFilter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using BenchmarkDotNet.Filters;
+using BenchmarkDotNet.Running;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BenchmarkDotNet.Extensions
+{
+    class ExclusionFilter : IFilter
+    {
+        private string filter;
+
+        public ExclusionFilter(string _filter)
+        {
+            if (!String.IsNullOrEmpty(_filter))
+            {
+                filter = _filter.ToLowerInvariant();
+            }
+        }
+
+        public bool Predicate(BenchmarkCase benchmarkCase)
+        {
+            if(String.IsNullOrEmpty(filter))
+            {
+                return true;
+            }
+            string testName = benchmarkCase.DisplayInfo.ToLowerInvariant().Substring(0, benchmarkCase.DisplayInfo.IndexOf(':') + 1);
+            return !testName.Contains(filter);
```

Comment:
the "inclusive" `--filter` allows for some simple regular expressions like `System.Collections*ToArray`, IMHO it would be great to keep consistency between both to avoid the user confusion

maybe you could even use [GlobFilter](https://github.com/dotnet/BenchmarkDotNet/blob/master/src/BenchmarkDotNet/Filters/GlobFilter.cs#L19-L29) from BDN and just negate what it is returning?
```cs
internal class ExclusionFilter : IFilter
{
    private readonly GlobFilter _inclusiveFilter;
    
    internal ExclusionFilter(string[] patterns) => _inclusiveFilter = new GlobFilter(patterns);
    
    public bool Predicate(BenchmarkCase benchmarkCase) => !_inclusiveFilter.Predicate(benchmarkCase);
}
```

Reply:
I can switch over to the GlobFilter, I was actually wanting to use it, but didn't know where the type was :)